app/config.py

```diff
@@ -380,6 +381,11 @@ class Config(object):
             "schedule": crontab(),
             "options": {"queue": QueueNames.PERIODIC},
         },
+        "run-generate-reports": {
+            "task": "run-generate-reports",
+            "schedule": crontab(),
+            "options": {"queue": QueueNames.PERIODIC},
```

Comment:
This is a task that can take quite some time if I am not mistaken? I wonder if we should locate it in a queue and tasks of its own, as this might slow down other periodic tasks when a report is generated. We have 4 workers per pod, and the primary pods where that would run can scale to 30, if I am not wrong.. but the time to scale up could slow down the other tasks. I would like to speak about this with you all and @ben851 .

Reply:
Yes the task could take up to 20 minutes for our biggest services. Good idea with the dedicated queue! Makes sense to me.